style.css

```diff
@@ -237,3 +267,245 @@ body {
 }
 
 /* End of menu */
+
+/* start of cards  */
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 12px;
+  width: 327px;
+  height: 552px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.portfolio {
+  min-width: 295px;
+  height: 220px;
+  border-radius: 8px;
+}
+
+.works {
+  display: grid;
+  row-gap: 88px;
+  align-items: center;
+  justify-content: center;
+  width: 372px;
+  padding-top: 114px;
+  background: #e5e5e5;
+}
+
+.title {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  height: 36px;
+  color: var(--title-color);
+}
+
+.subtitle {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+  margin-right: 12px;
+}
+
+.experience {
+  display: flex;
+  margin-top: 20px;
+}
+
+.block {
+  display: flex;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.card_body {
+  min-width: 295px;
+}
+
+.content {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  margin-top: 20px;
+}
+
+.skills {
+  list-style: none;
+  display: flex;
+  margin-top: 12px;
+}
+
+.button {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  justify-content: center;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  border: 1px solid #7f8cff;
+  border-radius: 8px;
+  background: #fff;
+  width: 124px;
+  height: 48px;
+  margin-top: 24px;
+}
```

Comment:
- [ ] As stated in the project requirement Kindly Implement the button interactions by following the button styling: Active, Hover and Pressed (see design) .

![Capture](https://user-images.githubusercontent.com/93598189/213205797-02aa9ed0-4a1a-4d1a-9e08-399dea0385e6.PNG)